TestApp/Enums.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestApp
+{
+    public enum FoodType
+    {
+        Pashtetik,
+        Kormik,
+        Smakolyk
+    }
+    public enum FoodSpeciality
+    {
+        Wet,
```

Comment:
add another enum value
e.g. 
Wet = 1,
Dry = 2

and use enum types using those values